app/models/bank_account.rb

```diff
@@ -21,15 +17,13 @@
 # Indexes
 #
 #  index_bank_accounts_on_hashed_account_number  (hashed_account_number)
-#  index_bank_accounts_on_hashed_routing_number  (hashed_routing_number)
 #  index_bank_accounts_on_intake_id              (intake_id)
 #
 # Foreign Keys
 #
 #  fk_rails_...  (intake_id => intakes.id)
 #
 class BankAccount < ApplicationRecord
-  self.ignored_columns = [:encrypted_routing_number, :encrypted_routing_number_iv, :hashed_routing_number, :_routing_number]
```

Comment:
I'm 49% tempted to say don't delete `ignored_columns`.

I vote 51% in favor of "it's OK to remove it". The database will be in a state where the columns are gone when the code is executed, so I guess it's OK to remove it!